book/05-reporting.Rmd

```diff
@@ -1 +1,164 @@
+
+- [xaringan](https://github.com/yihui/xaringan): For HTML presentations via [remark.js](https://remarkjs.com).
+
+- [blogdown](https://github.com/rstudio/blogdown): For creating websites. Example: [https://pjs-web.de/](https://pjs-web.de/)
+
+- [rticles](https://github.com/rstudio/rticles): For scientific paper writing in R.
+
+### Code chunks in R Markdown
+
+To insert code into an R Markdown document, one needs to add a so called "code chunk".
+
+````
+```{<language>}
+
+```
+````
+
+This tells the document that everything within the three backticks should be interpreted as code using the given language.
+Code can be shown/hidden, evaluation can be prevented on demand, results can be cached, etc.
+See https://yihui.org/knitr/options/ for a full list of supported options
+
+### R Notebooks
+
+[R Notebooks](https://bookdown.org/yihui/rmarkdown/notebook.html) are a special form of the `html_document`.
+To use it, specify `html_notebook` as the "output" type in the YAML header.
+This output format was created by RStudio as an alternative to `html_document`.
+
+Differences compared to `html_document`:
+
+- Code output is shown inside the editor and not in the console (can be changed)
+
+- Instant preview of the output document without having to get all the code in the document running. The results from the last successful code execution will be used (if there was one).
+
+- Link in the HTML doc to download the source `.Rmd` file
+
+- Option to toggle on/off code chunks for the whole document
+
+- Output file extension is named `.nb.html`
+
+### Workflow
+
+R Markdown documents are most often used for reporting of results created in an Rscript.
+This enables a seamless integration of data processing tasks into the subsequent reporting.
+
+Reporting often splits up into different formats:
+
+- Talks using presentation slides ([xaringan](https://bookdown.org/yihui/rmarkdown/xaringan.html), [ioslides](https://bookdown.org/yihui/rmarkdown/ioslides-presentation.html), [Slidy](https://bookdown.org/yihui/rmarkdown/slidy-presentation.html), [Beamer](https://bookdown.org/yihui/rmarkdown/beamer-presentation.html), [Powerpoint](https://bookdown.org/yihui/rmarkdown/powerpoint-presentation.html))
+- written reports (Word, PDF), possibly using LaTeX input
+
+R objects (containing results) can directly be used in the reports to present the results (data, plots).
+If the complete workflow of an analysis has been setup in R, changes at certain stages of the workflow (e.g. incoming data) can easily be integrated.
+
+This is the point where packages like [drake](https://docs.ropensci.org/drake/), [workflowr](https://jdblischak.github.io/workflowr/) and [rrtools](https://github.com/benmarwick/rrtools) jump in to simplify reproducible workflows in R.
+
+A widely used concept is to start a project following the structure of an R package.
```

Comment:
An existing section is dedicated to this.

Reply:
I currently do not see one (neither in the deployed book nor in the existing files).

If you tell me, I can add a cross-ref.